src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationService.kt

```diff
@@ -644,39 +669,18 @@ class AdjudicationService(
         chargeSequence = chargeSequence,
         incidentCharge = incidentCharge,
         offence = incidentCharge.offence,
-      ).let { adjudicationHearingResultRepository.save(it) }
+      ).let { adjudicationHearingResultRepository.saveAndFlush(it) }
         .also {
+          telemetryMap["resultSequence"] = it.id.resultSequence.toString()
           telemetryClient.trackEvent(
             "hearing-result-created",
-            mapOf(
-              "adjudicationNumber" to adjudicationNumber.toString(),
-              "chargeSequence" to chargeSequence.toString(),
-              "hearingId" to hearingId.toString(),
-              "resultSequence" to "1",
-            ),
+            telemetryMap,
             null,
           )
         }
-    } else {
-      telemetryClient.trackEvent(
```

Comment:
don't need the ignore now and update service is mapping "REF_INAD" finding codes